.github/workflows/check-releases-archives.yaml

```diff
@@ -0,0 +1,76 @@
+name: Check the releases' archives
+
+on:
+  schedule:
+    # Run every two days at midnight
+    - cron: "0 0 */2 * *"
+  workflow_dispatch:
+
+jobs:
+  infer-releases:
+    runs-on: ubuntu-latest
+    name: "Infer releases to check"
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.maintained-releases }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - id: set-matrix
+        name: Infer maintained releases
+        run: |
+          set -x
+
+          all_branches="$(git branch --all | grep -E "remotes.*ubuntu-[0-9]{2}\.[0-9]{2}$" | cut -d '/' -f 3)"
+
+          maintained=()
+
+          for branch in $all_branches; do
+            echo "Checking maintenance status for '${branch}'"
+            git switch $branch
+            
+            eol="$(yq .maintenance."end-of-life" chisel.yaml || echo null)"
+            if [[ "$eol" != "null" && "$(date +"%Y-%m-%d")" < "$eol" ]];
```

Comment:
note that this will trigger for unstable branches too. just an observation though. since this is not failing any PRs, lets merge as is and then observe closer to resolute.